examples/video-chat/ios/Podfile.lock

```diff
@@ -1425,6 +1419,6 @@ SPEC CHECKSUMS:
   Yoga: 64cd2a583ead952b0315d5135bf39e053ae9be70
   ZXingObjC: 8898711ab495761b2dbbdec76d90164a6d7e14c5
 
-PODFILE CHECKSUM: 0376e85da25bcfda7c0095d7c9d174ee924ff49d
+PODFILE CHECKSUM: 2c30f3251567073d449ed093b86832597a8e3058
 
-COCOAPODS: 1.14.3
+COCOAPODS: 1.15.2
```

Comment:
I've added task to lock this version in our repo at some point in future [FCE-253]